expression/expression.go

```diff
@@ -115,7 +175,7 @@ func getExpression(triggerExpression *TriggerExpression) (*govaluate.EvaluableEx
 			return nil, err
 		}
 
-		return validateUserExpression(triggerExpression, userExpression)
+		return validateExpressionVariables(triggerExpression, userExpression)
```

Comment:
After you patched expr by replacing them with Get, expr won't run all the arguments through Get? Because if it does, then there is no point in validating the arguments a second time

Reply:
there are two ways:
- if the `expr.Run()` is removed, then we will still need to check for the expression variables, because `expr` checks for this variables when Running.
- if the `expr.Run()` is kept, we can totally remove the variable checking logic.

I prefer the second option, what do you think about this?

Reply:
If `expr.Run()` checks all variables at all, even those in other branches, then this is a good option and there is no need to recheck them